pkg/genericapiserver/genericapiserver.go

```diff
@@ -585,25 +585,26 @@ func verifyEtcdServersList(options *options.ServerRunOptions) {
 	}
 }
 
-func verifySecurePort(options *options.ServerRunOptions) {
+func verifySecureAndInsecurePort(options *options.ServerRunOptions) {
```

Comment:
I thought we may add other port related flags in the future, so I used the name `verifySecureAndInsecurePort` to be clear. But it's reasonable to put all port flags verifying in the same func, such as the name you suggest. Thanks @lavalamp !